VMEncryption/main/Utils/HandlerUtil.py

```diff
@@ -403,72 +408,108 @@ def redo_current_status(self):
                               stat[0]["status"]["code"],
                               stat[0]["status"]["formattedMessage"]["message"])
 
+    def reject_settings(self):
```

Comment:
a small comment on why this method was introduced and in what scenarios it is invoked would be helpful

Reply:
Will do.

Reply:
Done